usr/share/rear/output/default/940_grub2_rescue.sh

```diff
@@ -15,28 +17,31 @@ is_true "$GRUB_RESCUE" || return 0
 # in particular do not run this script when GRUB Legacy is used
 # (for GRUB Legacy output/default/940_grub_rescue.sh is run):
 if [[ ! $( type -p grub-probe ) && ! $( type -p grub2-probe ) ]] ; then
-    LogPrint "Skipping GRUB_RESCUE setup for GRUB 2 (no GRUB 2 found)."
-    return
+    LogPrint "Cannot do GRUB_RESCUE setup: GRUB 2 not found (neither grub-probe nor grub2-probe)"
+    return 1
```

Comment:
Should it return 1 ? I think that systems with GRUB Legacy are supported and on those taking this branch is expected (they are handled by 940_grub_rescue.sh in the same directory).

Reply:
@pcahyna 
thank you for your review.
You are right. It should not even alert the user at all here
when output/default/940_grub_rescue.sh is responsible.